client/Dockerfile_calculator

```diff
@@ -0,0 +1,13 @@
+FROM node:current-alpine
```

Comment:
FWIW, I think the prevailing style is `calculator.Dockerfile`, not `Dockerfile_calculator`.

Reply:
done